packages/electron/README.md

```diff
@@ -0,0 +1,504 @@
+    // Submission url
+    // <universe> is the subdomain of your Backtrace instance (<universe>.backtrace.io)
+    // <token> can be found in Project Settings/Submission tokens
+    url: 'https://submit.backtrace.io/<universe>/<token>/json',
+};
+
+// Initialize the client with the options
+const client = BacktraceClient.initialize(options);
+
+// By default, Backtrace will send an error for Uncaught Exceptions and Unhandled Promise Rejections
+
+// Manually send an error
+client.send(new Error('Something broke!'));
+```
+
+### Integrate the SDK in the renderer process
+
+You can use any browser based Backtrace SDK in your renderer process, such as `@backtrace/browser` or
+`@backtrace/react`. Make sure to install it aside `@backtrace/electron`:
+
+```
+$ npm install @backtrace/react
+```
+
+To add Backtrace to the renderer process with Electron IPC support, you need to use a
+[preload script](https://www.electronjs.org/docs/latest/tutorial/tutorial-preload) in your `BrowserWindow`. Once you
+have one, add this import to it:
+
+```ts
+import '@backtrace/electron/main/preload';
+```
+
+This will ensure that IPC APIs that Backtrace uses are loaded correctly.
+
+After that, you can initialize Backtrace in your renderer process. There are two things that differ from the normal SDK
+initialization:
+
+1. URL is not used - you can leave it as an empty string,
+1. You need to use `addBacktraceElectron` with the builder.
+
+For example, if you are initializing the SDK using the `initialize` function:
+
+```ts
+const client = BacktraceClient.initialize(
+    {
+        url: '',
+        name: 'My Renderer App',
+        version: '0.0.1',
+    },
+    (builder) => addBacktraceElectron(builder),
+);
+```
+
+Or, if you are using the `build` function:
+
+```ts
+const builder = BacktraceClient.builder({
+    url: '',
+    name: 'My Renderer App',
+    version: '0.0.1',
+});
+
+addBacktraceElectron(builder);
+
+const client = builder.build();
+```
+
+#### Synchronized initialization
+
+When adding Electron to the renderer SDK, data is exchanged with the main process to ensure same session IDs are used.
+By default, this is done asynchronously as soon as possible. This may however be not enough in case of startup crashes
+just after the SDK initialization, because the data may not be exchanged yet.
+
+In those cases, you can use `synchronize` option in `addBacktraceBuilder`. Note, that if you do not ensure that the SDK
+in the main process is initialized by the time you call `addBacktraceBuilder`, **your application may hang with a white
+screen**. This is due to the synchronous nature of the IPC, that is blocking the renderer process until it receives a
+response.
+
+To enable synchronous initialization:
+
+```ts
+const client = BacktraceClient.initialize(
+    {
+        url: '',
+        name: 'My Renderer App',
+        version: '0.0.1',
+    },
+    (builder) => addBacktraceElectron(builder, { synchronous: true }),
+);
+```
+
+### Upload source maps
```

Comment:
I think we also need to add information about crash symbolication. I think we can do it outside this pull request but I'm pointing this out to not forget about this